content/tutorials/webrtc/datachannels/en/index.html

```diff
@@ -57,93 +57,92 @@
+
+<ul>
+  <li><b>Reliable and ordered mode guarantees the transmission of messages and also the order in which they are delivered</b>. This takes extra overhead, thus potentially making this mode slower. </li>
+  <li><b>Unreliable and unordered mode does not guarantee every message gets to the other side nor what order they get there in</b>. This removes the overhead, allowing this mode to work much faster.</li>
+  <li><b>Partial reliable mode guarantees the transmission of message under a specific condition, such as a retransmit timeout or a maximum amount of retransmissions</b>. The ordering of messages is also configurable.
+</ul>
+
+<p>Performance for the first two modes is about the same when there are no packet losses. However, in reliable and ordered mode, a lost packet causes other packets to get blocked behind it, and the lost packet might be stale by the time that it is retransmitted and arrives. It is, of course, possible to use multiple data channels within the same app, each with their own reliable or unreliable semantics.</p>
+
+<p>Here's a helpful table from <a href="https://www.oreilly.com/library/view/high-performance-browser/9781449344757/">High Performance Browser Networking</a> by <a href="https://www.igvita.com/" title="Ilya Grigorik's website">Ilya Grigorik</a>:</p>
 
 <table>
 <th>
   <td style="font-weight: bold;">TCP</td><td style="font-weight: bold;">UDP</td><td style="font-weight: bold;">SCTP</td>
 </th>
 <tr>
-  <td style="font-weight: bold;">Reliability</td><td>reliable</td><td>unreliable</td><td>configurable</td>
+  <td style="font-weight: bold;">Reliability</td><td>Reliable</td><td>Unreliable</td><td>Configurable</td>
 </tr>
 <tr>
-  <td style="font-weight: bold;">Delivery</td><td>ordered</td><td>unordered</td><td>configurable</td>
+  <td style="font-weight: bold;">Delivery</td><td>Ordered</td><td>Unordered</td><td>Configurable</td>
 </tr>
 <tr>
-  <td style="font-weight: bold;">Transmission</td><td>byte-oriented</td><td>message-oriented</td><td>message-oriented</td>
+  <td style="font-weight: bold;">Transmission</td><td>Byte-oriented</td><td>Message-oriented</td><td>Message-oriented</td>
 </tr>
 <tr>
-  <td style="font-weight: bold;">Flow control</td><td>yes</td><td>no</td><td>yes</td>
+  <td style="font-weight: bold;">Flow control</td><td>Yes</td><td>No</td><td>Yes</td>
 </tr>
 <tr>
-  <td style="font-weight: bold;">Congestion control</td><td>yes</td><td>no</td><td>yes</td>
+  <td style="font-weight: bold;">Congestion control</td><td>Yes</td><td>No</td><td>Yes</td>
 </tr>
 </table>
 
-<p>(From <a href="http://chimera.labs.oreilly.com/books/1230000000545/ch18.html">High Performance Browser Networking</a> by <a href="http://www.igvita.com/" title="Ilya Grigorik's website">Ilya Grigorik</a>.)</p>
-
-<p>RTCDataChannel can work in unreliable and unordered mode (analogous to User Datagram Protocol or UDP), reliable and ordered mode (analogous to Transmission Control Protocol or TCP) and partial reliable modes:</p>
```

Comment:
These deleted paragraphs should be reinstated :).

Reply:
These are still in the article, but I moved them a little higher.